spec2sdk/models/entities.py

```diff
@@ -0,0 +1,323 @@
+import textwrap
+from abc import abstractmethod
+from pathlib import Path
+from typing import Any, Sequence
+
+from spec2sdk.base import Model
+from spec2sdk.models.imports import Import
+from spec2sdk.templating import create_jinja_environment
+
+
+class PythonType(Model):
+    name: str | None
+    description: str | None
+    default_value: Any
+
+    @property
+    @abstractmethod
+    def type_hint(self) -> str: ...
+
+    @property
+    @abstractmethod
+    def imports(self) -> Sequence[Import]: ...
+
+    @abstractmethod
+    def render(self) -> str: ...
```

Comment:
This code be a property maybe? Something like `code` or some other name maybe?

Reply:
I would keep it as a method because in some cases we can render Jinja templates. On the contrary, properties usually do some simple computations. I could use caching property but I don't want to do this is this case. From my point of view, using method instead of property in this case shows that it hides intensive process and shouldn't be called many times without thinking about performance.